core/src/main/java/com/nerds/gamejam/ecs/system/RenderSystem.java

```diff
@@ -49,18 +45,21 @@ protected void begin() {
 
     @Override
     protected void process(int e) {
```

Comment:
Does artemis make a guarantee over entity order wrt processing? ie will background image always render 1st due to its creation being first?

Reply:
I'm not sure tbh, unless artemis starts to reuse the ids then we should be fine. I'd be surprised if artemis reuses ids before running out, seems unlikely we'd hit a limit of any kind with the game we've got planned

Reply:
I was more meaning if you created two entities that are given ids 1and 2 will they be processed in order 1 then 2

Reply:
:( unfortunately artemis is super efficient! If you remove an entity is takes the last id in the idBag and puts it into the index of the removed entityId. I've added a layer to texture, then the rendering just happens in layers. So, we can guarantee layer order but not the render order within layers